### PR TITLE
IT Picture Updates

### PR DESCRIPTION
**Pull request by gabrielcossette**

### CODE REVIEW

File: _pages/en/it-picture-long-term.md [hunk 1]
```diff
@@ -119,56 +126,57 @@ As a result:
...
 
 Teams work using the agile methodology to gather and implement customer feedback, working in small batches (iteration or sprint).
 Where possible work is being done in the open make the code, data and information available to other teams and interested parties.
 
 ##### Internal IT services
 
-IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
-IITB is responsible for providing the automated services required to run a secure and agile business.
-This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment by which business may securely deploy their own solutions.
+IT is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
```
Comment (remyb2canada) [inline _pages/en/it-picture-long-term.md]: That should be IITB. Internal IT services would remain in a corporate IT branch.

File: _pages/en/it-picture-long-term.md [hunk 2]
```diff
@@ -119,56 +126,57 @@ As a result:
...
+IT is responsible for providing the automated services required to run a secure and agile business.
+This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment in which business may securely deploy their own solutions.
 
 #### Strong Culture of Learning
 
-IITB has a very open culture that values learning (TODO: find a tangible example)
-TODO: Add recruitment best practices, including for domains other than IT.
-ESDC-IITB services have performance metrics that are used to continuously improve. These include most committees as well.
-TODO: Add recruitment best practices, including for domains other than IT.
+- IT has a very open culture that values learning
```
Comment (remyb2canada) [inline _pages/en/it-picture-long-term.md]: Change "IT" to "ESDC"

File: _pages/en/it-picture-long-term.md [hunk 3]
```diff
@@ -119,56 +126,57 @@ As a result:
...
+  - TODO: find a tangible example
+- ESDC IT services have performance metrics that are used to continuously improve. These include most committees as well.
```
Comment (remyb2canada) [inline _pages/en/it-picture-long-term.md]: "ESDC IT services"... Do we still mean the corporate IT branch that is IITB?

File: _pages/en/it-picture-long-term.md (hunk 1 above)
Comment (smellems) [reply _pages/en/it-picture-long-term.md]: There is currently no mention of IITB in the long term vision.  It's referred to only as IT or IT services.

File: _pages/en/enable-learning.md [hunk 4]
```diff
@@ -15,17 +23,41 @@ Activities which align to this include, though are not limited to
...
+
+- Formal Training (e.g. classroom)
+- Informal Learning
+
+Formal training being a mature and well-recognized practice, ESDC will need to focus on understanding and encouraging informal learning among employees.
+Especially as [recent estimates suggest](https://www.groupoe.com/images/Accelerating_On-the-Job-Learning_-_White_Paper.pdf) that between 70% to 80% of job-related knowledge isn't learned in formal training but rather **informally** on-the-job.
+We observe that formal training hasn't been designed to provide employees with the "on demand" learning they need to maintain current knowledge and proficiency.
+
+> **Informal learning** is any learning that is not formal learning or non-formal learning, such as self-directed learning or learning from experience. [...] it has no set objective in terms of learning outcomes and is never intentional from the learner's standpoint. [...] this includes heuristic language building, socialization, enculturation, and play.
+> Source: [Wikipedia](https://en.wikipedia.org/wiki/Informal_learning)
```
Comment (gcharest) [inline _pages/en/enable-learning.md]: Lien vers Wikipédia?

Comment (gabrielcossette) [reply _pages/en/enable-learning.md]: C'est la définition de Wikipédia. Je croyais bon de l'inclure.

File: _pages/en/it-picture-medium-term.md [hunk 5]
```diff
@@ -178,53 +176,62 @@ TODO: Find cloud adoption rates | Duplicate from previous section?
...
 
 - Corporate services (hardware and software support)
 - Cyber security
 - Technology architecture
 - Development teams
 - Technology Infrastructure Management
 
-Corporate Services includes the establishment and promotion of sound Information Management (IM) structure and culture. Information is seen/viewed/treated as a corporate asset and managed through a centralized Records Management System where information is easily searchable, secure and not duplicated.
+[IITB`s Way Forward](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Documents/Departmental%20IMIT%20Plan/19-20%20Plans/IITB%20Moving%20Forward%20v2.docx) enabled IITB to provide ESDC with modern Corporate Services which includes the establishment and promotion of sound Information Management (IM) structure and culture. Information is seen/viewed/treated as a corporate asset and managed through a Records Management ecosystem where information is easily searchable, secure and not duplicated. IITB fully supports the release of open datasets. Also, collaboration platforms are easily accessible and linked between each other to provide a Digital Workspace experience.
```
Comment (gcharest) [inline _pages/en/it-picture-medium-term.md]: We probably need to review this section to have 1 sentence per line (easier tracking of changes afterwards)

File: _pages/en/it-picture-medium-term.md [hunk 6]
```diff
@@ -178,53 +176,62 @@ TODO: Find cloud adoption rates | Duplicate from previous section?
...
+The IT environment is patched in a timely manner with minimal impact to staff productivity.
 
-- TODO: Validate employee mobility with trends
+TODO: Automation, multiple trusted sources to assure tools are kept up to date
 
-TODO: Patching, automation, multiple trusted sources to assure tools are kept up to date
+IITB has provided ESDC with Microsoft Azure's Office 365 in a way that follows Information Management policy requirements
 
-TODO: Different work profiles = different risk mitigation (e.g. open web services)
+IT Policies have been refreshed to align with the Policy on Service and Digital, framing employee behaviors for a modern and open organization.
```
Comment (gcharest) [inline _pages/en/it-picture-medium-term.md]: "[...], aligned with the rest of the GC."

**Comment by gcharest** [comment]
Finished pushing last changes.

**Pull request merged**